### PR TITLE
Bring Tailwind CSS example in line with official Tailwind docs

### PR DESCRIPTION
**Pull request by ncarchedi**
Accompanying the release of Tailwind CSS v2, @adamwathan recently added [detailed instructions](https://tailwindcss.com/docs/guides/nextjs) for integrating with Next to the official Tailwind docs. This PR brings the Tailwind example in line with those instructions in a minimal way by:

1. Creating a new app with `create-next-app`
2. Adding Tailwind per the instructions
3. Updating `index.js` to use Tailwind for styling (screenshot below)
4. Removing other CSS cruft

*Note: I realize someone has already updated this example to use Tailwind v2, but it still deviates in small ways from what you'd get if you followed the Tailwind docs verbatim. Not a huge deal, but could be confusing to newcomers.*

Screen Shot 2020-12-02 at 8 20 48 AM

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| buildDuration | 9.1s | 9.3s | ⚠️ +230ms |
| nodeModulesSize | 82.3 MB | 82.3 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.88 | 1.881 | 0 |
| / avg req/sec | 1329.68 | 1329.16 | ⚠️ -0.52 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.187 | 1.198 | ⚠️ +0.01 |
| /error-in-render avg req/sec | 2105.93 | 2087.02 | ⚠️ -18.91 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..8b6e.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-3c9ff84..1d7c.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fca3d..2eb1.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-db223d9..dbd7.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 8.01 kB | 8.01 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 620 B | 620 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| buildDuration | 10.9s | 10.8s | -26ms |
| nodeModulesSize | 82.3 MB | 82.3 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..8b6e.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-3c9ff84..1d7c.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fca3d..2eb1.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-db223d9..dbd7.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 8.01 kB | 8.01 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _error.js | 997 kB | 997 kB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 998 kB | 998 kB | ✓ |
| link.js | 1.05 MB | 1.05 MB | ✓ |
| routerDirect.js | 1.04 MB | 1.04 MB | ✓ |
| withRouter.js | 1.04 MB | 1.04 MB | ✓ |
| Overall change | 5.14 MB | 5.14 MB | ✓ |

</details>

</details>
Commit: 85a57a25ee2c15914f23ea082c9ad66723159db3

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| buildDuration | 8.3s | 8.7s | ⚠️ +406ms |
| nodeModulesSize | 82.3 MB | 82.3 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.732 | 1.722 | -0.01 |
| / avg req/sec | 1443.13 | 1451.74 | +8.61 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.029 | 1.066 | ⚠️ +0.04 |
| /error-in-render avg req/sec | 2428.82 | 2344.42 | ⚠️ -84.4 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..8b6e.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-3c9ff84..1d7c.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fca3d..2eb1.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-db223d9..dbd7.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 8.01 kB | 8.01 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 620 B | 620 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| buildDuration | 10s | 10.1s | ⚠️ +100ms |
| nodeModulesSize | 82.3 MB | 82.3 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..8b6e.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-3c9ff84..1d7c.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59 kB | 59 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fca3d..2eb1.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-db223d9..dbd7.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 8.01 kB | 8.01 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _error.js | 997 kB | 997 kB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 998 kB | 998 kB | ✓ |
| link.js | 1.05 MB | 1.05 MB | ✓ |
| routerDirect.js | 1.04 MB | 1.04 MB | ✓ |
| withRouter.js | 1.04 MB | 1.04 MB | ✓ |
| Overall change | 5.14 MB | 5.14 MB | ✓ |

</details>

</details>
Commit: ef845d33f1986f80220d2ed15f0a82e1567984d7

File: examples/with-tailwindcss/tailwind.config.js [hunk 1]
```diff
@@ -1,12 +1,8 @@
 module.exports = {
-  purge: ['./components/**/*.{js,ts,jsx,tsx}', './pages/**/*.{js,ts,jsx,tsx}'],
-  darkMode: 'media', // 'media' or 'class'
+  purge: ['./pages/**/*.js', './components/**/*.js'],
```
Comment (timneutkens) [inline examples/with-tailwindcss/tailwind.config.js:2]:
```suggestion
  purge: ['./pages/**/*.{js,ts,jsx,tsx}', './components/**/*.{js,ts,jsx,tsx}'],
```

Comment (timneutkens) [reply examples/with-tailwindcss/tailwind.config.js]: Opened a PR for the tailwind docs here: https://github.com/tailwindlabs/tailwindcss.com/pull/727

**Review by timneutkens — DISMISSED** [review]
Thanks! Looks good now 👍

**Review by timneutkens — DISMISSED** [review]

**Comment by ijjk** [comment]
## Failing test suites 
Commit: e398ad42c44f2454e0e25b81ba7d53e572baefad

**test/integration/dist-dir/test/index.test.js** 
- distDir > should handle null/undefined distDir

<details>
<summary>Expand output</summary>

  ● distDir › should handle null/undefined distDir

    expect(received).toBe(expected) // Object.is equality

    Expected: 0
    Received: 88

      63 |     await fs.writeFile(nextConfig, origNextConfig)
      64 | 
    > 65 |     expect(stderr.length).toBe(0)
         |                           ^
      66 |   })
      67 | })
      68 | 

      at Object.<anonymous> (integration/dist-dir/test/index.test.js:65:27)

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| buildDuration | 9.4s | 9.4s | ⚠️ +21ms |
| nodeModulesSize | 73.3 MB | 73.3 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.851 | 2.054 | ⚠️ +0.2 |
| / avg req/sec | 1350.92 | 1217.36 | ⚠️ -133.56 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.164 | 1.238 | ⚠️ +0.07 |
| /error-in-render avg req/sec | 2147.24 | 2018.67 | ⚠️ -128.57 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..38ed.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-c71ae4..70cd.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-7faf09b..eba4.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| index.html gzip | 614 B | 614 B | ✓ |
| link.html gzip | 620 B | 620 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| buildDuration | 11s | 11s | ⚠️ +20ms |
| nodeModulesSize | 73.3 MB | 73.3 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| 677f882d2ed8..38ed.js gzip | 13.1 kB | 13.1 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.4 kB | 59.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.3 kB | 31.3 kB | ✓ |
| Overall change | 31.3 kB | 31.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _app-2a09aa2..4a98.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-8b758..aef6.js gzip | 3.46 kB | 3.46 kB | ✓ |
| hooks-c71ae4..70cd.js gzip | 887 B | 887 B | ✓ |
| index-bbee2f..528b.js gzip | 227 B | 227 B | ✓ |
| link-7faf09b..eba4.js gzip | 1.64 kB | 1.64 kB | ✓ |
| routerDirect..bf84.js gzip | 303 B | 303 B | ✓ |
| withRouter-a..5826.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.09 kB | 8.09 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 321 B | 321 B | ✓ |
| Overall change | 321 B | 321 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.19 MB | 5.19 MB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Webpack 5 Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| buildDuration | 11.5s | 11.6s | ⚠️ +112ms |
| nodeModulesSize | 73.3 MB | 73.3 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 1.877 | 1.871 | -0.01 |
| / avg req/sec | 1331.96 | 1336.44 | +4.48 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.18 | 1.204 | ⚠️ +0.02 |
| /error-in-render avg req/sec | 2119.01 | 2077.26 | ⚠️ -41.75 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| 597-c48889ef..1870.js gzip | 13 kB | 13 kB | ✓ |
| framework.HASH.js gzip | 39.3 kB | 39.3 kB | ✓ |
| main-HASH.js gzip | 6.58 kB | 6.58 kB | ✓ |
| webpack-HASH.js gzip | 954 B | 954 B | ✓ |
| Overall change | 59.8 kB | 59.8 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31.1 kB | 31.1 kB | ✓ |
| Overall change | 31.1 kB | 31.1 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _app-a5b2e84..33cf.js gzip | 1.26 kB | 1.26 kB | ✓ |
| _error-6f1f9..5577.js gzip | 3.38 kB | 3.38 kB | ✓ |
| hooks-725de8..0a15.js gzip | 904 B | 904 B | ✓ |
| index-939503..6e1c.js gzip | 232 B | 232 B | ✓ |
| link-e0cc871..fdbb.js gzip | 1.63 kB | 1.63 kB | ✓ |
| routerDirect..9360.js gzip | 308 B | 308 B | ✓ |
| withRouter-6..44ec.js gzip | 304 B | 304 B | ✓ |
| Overall change | 8.02 kB | 8.02 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| _buildManifest.js gzip | 299 B | 299 B | ✓ |
| Overall change | 299 B | 299 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ncarchedi/next.js canary | Change |
| - | - | - | - |
| index.html gzip | 588 B | 588 B | ✓ |
| link.html gzip | 593 B | 593 B | ✓ |
| withRouter.html gzip | 581 B | 581 B | ✓ |
| Overall change | 1.76 kB | 1.76 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -43,7 +43,7 @@
         "props": { "pageProps": {} },
         "page": "/",
         "query": {},
-        "buildId": "NoCMOHEFQvgr4ThCbnbAL",
+        "buildId": "DTTzyN3QDZ3T1oRZy_9wR",
         "isFallback": false,
         "gip": true
       }
@@ -77,11 +77,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/NoCMOHEFQvgr4ThCbnbAL/_buildManifest.js"
+      src="/_next/static/DTTzyN3QDZ3T1oRZy_9wR/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/NoCMOHEFQvgr4ThCbnbAL/_ssgManifest.js"
+      src="/_next/static/DTTzyN3QDZ3T1oRZy_9wR/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -48,7 +48,7 @@
         "props": { "pageProps": {} },
         "page": "/link",
         "query": {},
-        "buildId": "NoCMOHEFQvgr4ThCbnbAL",
+        "buildId": "DTTzyN3QDZ3T1oRZy_9wR",
         "isFallback": false,
         "gip": true
       }
@@ -82,11 +82,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/NoCMOHEFQvgr4ThCbnbAL/_buildManifest.js"
+      src="/_next/static/DTTzyN3QDZ3T1oRZy_9wR/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/NoCMOHEFQvgr4ThCbnbAL/_ssgManifest.js"
+      src="/_next/static/DTTzyN3QDZ3T1oRZy_9wR/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -43,7 +43,7 @@
         "props": { "pageProps": {} },
         "page": "/withRouter",
         "query": {},
-        "buildId": "NoCMOHEFQvgr4ThCbnbAL",
+        "buildId": "DTTzyN3QDZ3T1oRZy_9wR",
         "isFallback": false,
         "gip": true
       }
@@ -77,11 +77,11 @@
       async=""
     ></script>
     <script
-      src="/_next/static/NoCMOHEFQvgr4ThCbnbAL/_buildManifest.js"
+      src="/_next/static/DTTzyN3QDZ3T1oRZy_9wR/_buildManifest.js"
       async=""
     ></script>
     <script
-      src="/_next/static/NoCMOHEFQvgr4ThCbnbAL/_ssgManifest.js"
+      src="/_next/static/DTTzyN3QDZ3T1oRZy_9wR/_ssgManifest.js"
       async=""
     ></script>
   </body>
```
</details>
</details>
Commit: e398ad42c44f2454e0e25b81ba7d53e572baefad

**Comment by dfelber** [comment]
Are the dependencies `react` and `react-dom` not required anymore? ;)

**Pull request merged**